### ISSUE TITLE
Sign up button does not work on Chrome iOS

### ISSUE THREAD

**Issue by grodeska** [civicmakers/client#188]
Reported by Danny Spitzberg on 2/21/16

**Comment by ybentz** [comment]
@grodeska I don't have an iphone to test this, can you try to reproduce it?

**Comment by grodeska** [comment]
Yep, I'm having the same problem on my iPhone. I click the "sign up" button
on the home page and nothing happens.

Clicking "add tool/project" takes me right to the form without any kind of
auth prompt.

Signup and add buttons are working on Safari on iOS

Also, this may be a separate issue, but I'm not seeing the header in in
Safari or Chrome on iOS. Screen shot attached.

[image: Inline image 1]

On Mon, Feb 22, 2016 at 3:22 PM, ybentz notifications@github.com wrote:

> @grodeska https://github.com/grodeska I don't have an iphone to test
> this, can you try to reproduce it?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/civicmakers/client/issues/188#issuecomment-187434073.

## 

Lawrence Grodeska
Founder & CEO
CivicMakers http://civicmakers.com
415-819-8495
